app/uk/gov/hmrc/apiplatformoutboundsoap/scheduled/LockedScheduledJob.scala

```diff
@@ -35,22 +36,18 @@ trait LockedScheduledJob {
 
   val releaseLockAfter: Duration
 
-  val lockRepository: LockRepository
+  val lockRepository: MongoLockRepository
 
-  lazy val lockKeeper = new LockKeeper {
-    override def repo: LockRepository = lockRepository
-    override def lockId: String  = s"$name-scheduled-job-lock"
-    override val forceLockReleaseAfter: Duration = releaseLockAfter
-  }
+  lazy val lockKeeper: LockService = LockService(lockRepository, lockId = s"$name-scheduled-job-lock", ttl = 1.hour)
 
-  def isRunning: Future[Boolean] = lockKeeper.isLocked
+  def isRunning: Future[Boolean] = lockRepository.isLocked(lockKeeper.lockId, "owner")
```

Comment:
I don't think this will work because the lock is created using a private owner ID 
https://github.com/hmrc/hmrc-mongo/blob/94f87658da8ad2c39e3ebabf9a13e251b0cd331c/hmrc-mongo-play-28/src/main/scala/uk/gov/hmrc/mongo/lock/LockService.scala#L33

Reply:
Hmm, I pulled it from the PPNS code that was post upgrade https://github.com/hmrc/push-pull-notifications-api/blob/592783d66ddff451fdc3eb4c630be03ab9e72cea/app/uk/gov/hmrc/pushpullnotificationsapi/scheduling/LockedScheduledJob.scala#L34